meetings/2022-12-14.md

```diff
@@ -69,11 +94,55 @@
+* SC: This is one of the steps just before writing a line of the spec. Just because someone proposed a use case, doesn't mean there has to be a requirement from it. It's to get a signal from the community that there is a significant amount of parties interested in a requirement for this use case. 
+* PA: I see in Issue #9 one issue document per spec, I'm not sure this is the right granularity. For example, for SPARQL, specifications are focused on a particular use case might be transversal, might gut feeling is it might be more efficient to have use cases pointing to the specification they touch on, not necessarily to segregate use cases per specification.
+* WT: +1
+* SC: That's interesting. The practice I was following was LDP working group.  I understand different groups may work differently. LDP UCR does not have a back reference to a requirement from the LDP spec. I don't know if another document somewhere that maps functional requirement #1 with a requirement in the spec. No link from either document to the other. Is that the connection you were looking for? UCR to requirement.
+* PA: My point was the scope of each specification might be too narrow to justify use cases about that specification. I would feel more comfortable with one use case document maybe not for whole Solid but for core portions. There is a [use case document](https://www.w3.org/TR/rdf-dawg-uc/) from RDF Access working group that produced SPARQL. They don't point to specific recommendations they produced, but they have one document for use cases that covers a number of specifications.
+* SC: Good point. I don't think we have that. The user stories come close to that as a collection of things. As you can see, only two or three attempts at having UCR documents in context of authorization and notifications. Anything from Solid Protocol wasn't though through more than institutional knowledge, derived from lessons learned from LDP.  Are you suggesting we should have one overarching use case document? For all work items in Solid ecosystem?
+* PA: Yeah, it's a gut feeling, it would make more sense.
+* WT: I think I second that. We do have two big files with use cases now but even in those, there are subdivisions into sub-specs, so why not make one file of it and have different sections and label accordingly. I think that would be nice.
+* SC: We could. Accidental bottom-up that we have general need for a component like authorizations or notifications, some spec is needed focused only on that. You can take a requirement like creating a container vice listening to updates to a container, they could go in same use case document, the approach here was that they are specific enough so you could have different solutions, so notifications could be a solution to listening and leave that in its own document. Since this is somewhat bottom-up, we could merge or create a document that touches these different areas. 
```

Comment:
```suggestion
* SC: We could. Accidental bottom-up that we have general need for a component like authorizations or notifications, some spec is needed focused only on that. You can take a requirement like creating a container by(???) listening to updates to a container, they could go in same use case document, the approach here was that they are specific enough so you could have different solutions, so notifications could be a solution to listening and left in its own document. Since this is somewhat bottom-up, we could merge or create a document that touches these different areas. 
```